Coding-Coach/coding-coach
# Remove horizontal scrollbar

# Details

## Description

There's a horizontal scrollbar on the app that we don't like. This gets rid of it!

## Steps to QA

1. Load the app.
2. Mac only: [turn on scrollbars temporarily](http://osxdaily.com/2011/08/03/show-scroll-bars-mac-os-x-lion/)
3. Verify there isn't a horizontal scrollbar.
4. 😎 

## Issue

Closes #273
